prime-database-backup/backup.sh

```diff
@@ -35,22 +35,17 @@ function databaseBackup() {
     # Backup PostGRES
```

Comment:
Can you please create README.md with key points for someone else to understand this backup system?

Reply:
For sure!